Signal/src/view controllers/CallViewController.swift

```diff
@@ -472,7 +473,28 @@ class CallViewController: UIViewController, CallDelegate {
         case .answering:
             return NSLocalizedString("IN_CALL_SECURING", comment: "Call setup status label")
         case .connected:
-            return NSLocalizedString("IN_CALL_TALKING", comment: "Call setup status label")
+            if let call = self.call {
+                let callDuration = call.connectionDuration()
+                let callDurationDate = Date(timeIntervalSinceReferenceDate:callDuration)
```

Comment:
I'm not sure that clang format supports swift. I believe the swift linter catches this though.